client/galaxy/scripts/mvc/tours.js

```diff
@@ -32,10 +32,15 @@ define(['libs/bootstrap-tour'],function(BootstrapTour) {
             }
             if (step.postclick){
                 step.onHide = function(){
-                    _.each(step.postclick, function(postclick){
```

Comment:
I agree on the replacement of the onHide with the onNext.
I was a bit hesitant on replacing it in the first place, since I thought we might want to have an onHide behavior that might differ in case an onPrev or an onNext was triggered, but I now think this is not the case. What do you think?